webdriver/tests/bidi/input/perform_actions/pointer_touch.py

```diff
@@ -195,10 +187,14 @@ async def test_touch_pointer_properties_tilt_twist(
         "pointerleave",
     ] == event_types
     assert events[2]["type"] == "pointerdown"
-    assert events[2]["tiltX"] == 21
-    assert events[2]["tiltY"] == -8
+    assert events[2]["altitudeAngle"] == 1.2
+    assert events[2]["azimuthAngle"] == 6
+    assert events[2]["tiltX"] == 20
+    assert events[2]["tiltY"] == -6
     assert events[2]["twist"] == 355
     assert events[3]["type"] == "pointermove"
+    assert events[3]["altitudeAngle"] == -23
+    assert events[3]["azimuthAngle"] == 61
```

Comment:
This should be `tiltX` and `tiltY`, right? Also I miss checks for both `azimuthAngle` and `altitudeAngle`.